File: CLRUSB/CLRUSB/PSE/CLR_PSE_Callbacks.vb
Namespace Global.PSE.CLR_PSE_Callbacks

    Public Delegate Sub Close() 'Logs+config
    'Logs
    Public Delegate Function Log_Open(logname As String) As Boolean
    Public Delegate Sub Log_Write(fmt As String)
    Public Delegate Sub Log_SetValue(par As Boolean)

    'USBCallback
    Public Delegate Sub CLR_CyclesCallback(cycles As Integer)

    'Config
    Public Delegate Function Config_Open(filename As String, writeAccess As Boolean) As Boolean
    Public Delegate Function Config_ReadInt(item As String, DefualtValue As Integer) As Integer
    Public Delegate Sub Config_WriteInt(item As String, Value As Integer)
End Namespace

File: CLRDEV9/CLRDEV9/PSE/CLR_PSE_Callbacks.cs
using System;
//using System.Collections.Generic;
//using System.Linq;
//using System.Text;
//using System.Threading.Tasks;

namespace PSE.CLR_PSE_Callbacks
{
    public delegate void Close();
    //Logs+config
    //Logs
    public delegate bool Log_Open(string logname);
    public delegate void Log_Write(string fmt);
    public delegate void Log_SetValue(bool par);

    //USBCallback
    public delegate void CLR_CyclesCallback(int cycles);

    //Config
    public delegate bool Config_Open(string filename, bool writeAccess);
    public delegate int Config_ReadInt(string item, int DefualtValue);
    public delegate void Config_WriteInt(string item, int Value);
}
